contrib/scripts/priv_validator_key-coping.json

```diff
@@ -0,0 +1,11 @@
+{
+  "address": "C913552357B2B69C728EB645B528C94A30DB7AB4",
+  "pub_key": {
```

Comment:
We can change the filename of `...key-coping.json` since that's a leftover from one of our old servers

Reply:
done at https://github.com/umee-network/umee/pull/1334/commits/eb589b271cdc86e5b594795abf462ada0a830c45